README.md

```
@@ -112,16 +133,16 @@ TalentMesh **solves these problems** by offering an **on-demand interview outsou
 We use various monitoring and visualization tools to track system health and metrics:  
 
-<h4> <b>Grafana</b> – Real-time application monitoring </h4> 
+<h4 id="grafana"> <b>Grafana</b> – Real-time application monitoring </h4> 
 <img src="public/images/grafana-dashboard.png" alt="Grafana Dashboard" width="100%">
 
-<h4> <b>Aspire</b> –  API and service management </h4> 
+<h4 id="aspire"> <b>Aspire</b> –  API and service management </h4> 
 <img src="public/images/aspire-logging-system.png" alt="Aspire Dashboard" width="100%">
 
-<h4> <b>RabbitMQ</b> – Queue management and message broker </h4>
+<h4 id="rabbitmq"> <b>RabbitMQ</b> – Queue management and message broker </h4>
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: h4

(MD033, no-inline-html)